build/scripts/Building.fsx

```diff
@@ -70,12 +72,8 @@ module Build =
         compileCore incremental
 
     let Clean() =
-        match (quickBuild, getBuildParam "target" = "clean") with
```

Comment:
Why was this removed? this section helped when running tests from the command line to rely on incremental builds. Incremental builds are not that fast on .NET core 1.1 but they still help a great deal.